### DISCUSSION TITLE
YUView by command line

### DISCUSSION THREAD

**Discussion by laidactienbn** [IENT/YUView#384] [Q&A] [answered]
I am using YUView on MacOS. I need to access to a file in /tmp but I cannot access /tmp using YUView client. I also cannot find any documents about using YUView client.

Is there any document for using YUView by command line?

**Comment by ChristianFeldmann** [comment, marked as answer]
Hi! Sorry I did not notice that there were discussion points being posted here. Maybe I should turn on notifications. 
There currently is nothing you can really do through the command line for YUView. But what you can do is provide a file path. YUView will then try to load it. E.g.: 
```
YUView /tmp/path/to/file.yuv
```
Another option is to create a playlist file and modify it with a text editor to add arbitrary paths. But the underlying problem you may have is that MacOS does not show all paths in the search dialog by default. But you can google how to enable this in MacOS.
